docs/programming-languages.md

```diff
@@ -50,6 +50,17 @@ to learn.
 
 [Get started with JavaScript](https://github.com/wasabifan/ev3dev-lang-js){: .btn .btn-default }
 
+## Java with LeJOS.
```

Comment:
"with" makes it sound like LeJOS is actually running. Could you say " Java (LeJOS-style)" or "Java (LeJOS-compatible)"?